workspaces/azure-devops/plugins/azure-devops-backend/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @backstage-community/plugin-azure-devops-backend
 
+## 0.8.0
+
+### Minor Changes
+
+- 0880746: **BREAKING** Removed support for what is known as the legacy backend, please use the New Backend System. Also removed the long deprecated `azureDevOps.token` configuration option, use `integrations.azure` instead.
```

Comment:
Minor breaking changes? Should this be a major?

Reply:
It's not 1.0 yet, I'm following this: https://github.com/backstage/backstage/blob/master/REVIEWING.md#reviewing-changeset-bump-levels